src/rebuild/srv.c

```diff
@@ -2384,11 +2383,11 @@ ds_rebuild_regenerate_task(struct ds_pool *pool, daos_prop_t *prop)
 		return DER_SUCCESS;
 	}
 
-	entry = daos_prop_entry_get(prop, DAOS_PROP_PO_SELF_HEAL);
-	D_ASSERT(entry != NULL);
-	if (entry->dpe_val & (DAOS_SELF_HEAL_AUTO_REBUILD | DAOS_SELF_HEAL_DELAY_REBUILD)) {
+	if (pool->sp_self_heal & (DAOS_SELF_HEAL_AUTO_REBUILD | DAOS_SELF_HEAL_DELAY_REBUILD) &&
```

Comment:
can the new function defined in pool.h be invoked here to perform the test?

Reply:
Sure.